students/hyska_monika/lesson_01_basics/Area_of_right_angled_triangle.py

```diff
@@ -0,0 +1,7 @@
+# Read the numbers b and h like this:
+
+a = int(input("set a: ")) #in formula for rectangle is used a not b
+h = int(input("set h: "))
+area = 0.5 * a * h
```

Comment:
This is necessary in most of the languages. Python (in version 3) handles `/` as floating point division already. That's why you used `//` in the apples exercise already.